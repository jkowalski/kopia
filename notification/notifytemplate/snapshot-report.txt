Subject: Kopia created {{ len .EventArgs.Snapshots }} snapshot{{ if gt (len .EventArgs.Snapshots) 1 }}s{{end}} on {{.Hostname}}

{{ range .EventArgs.Snapshots | sortSnapshotManifestsByName}}Path: {{ .Source.Path }}

  Status: {{ .StatusCode }}
  Start: {{ .StartTimestamp | formatTime  }} Duration: {{ .Duration }}
  Size: {{ .TotalSize | bytes }}, {{ .TotalFiles }} files, {{ .TotalDirs }} directories.
{{ if .Error }}  Error: {{ .Error }}
{{ end }}{{ if .RootEntry }}{{ if .RootEntry.DirSummary }}{{ if .RootEntry.DirSummary.FailedEntries }}
  Failed Entries:
{{ range .RootEntry.DirSummary.FailedEntries }}
  - {{.EntryPath}}: {{.Error}}{{ end }}{{ end }}{{ end }}
{{ end }}
{{ end }}Generated at {{ .EventTime | formatTime }} by Kopia {{ .KopiaBuildVersion }}.

https://kopia.io/