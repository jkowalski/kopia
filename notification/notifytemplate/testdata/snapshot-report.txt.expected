Subject: Kopia created 2 snapshots on some-host

Path: /some/other/path

  Status: fatal
  Start: 1969-12-31 16:00:00 -0800 PST Duration: 0s
  Size: 0 B, 0 files, 0 directories.
  Error: some top-level error

Path: /some/path

  Status: ok
  Start: 2020-01-01 19:04:05 -0800 PST Duration: 1.000000001s
  Size: 456 B, 123 files, 33 directories.

  Failed Entries:

  - /some/path: some error
  - /some/path2: some error

Generated at 2020-01-02 03:04:05.000000006 +0000 UTC by Kopia v0-unofficial.

https://kopia.io/